src/com/zone5cloud/core/Z5ErrorItem.java

```diff
@@ -1,10 +1,27 @@
 package com.zone5cloud.core;
 
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
 public class Z5ErrorItem {
+	public static class Metadata {
```

Comment:
Should this be a class or another type?

Would we want to create metadata in the future not related to terms?